### DISCUSSION TITLE
Internal notes field for experiment

### DISCUSSION THREAD

**Discussion by drikusroor** [Amsterdam-Music-Lab/MUSCLE#864] [Ideas]
In [this issue](https://github.com/Amsterdam-Music-Lab/MUSCLE/issues/861) that was created by Mariëlle today, it was (and is still) very hard for me to figure out about which experiment(s) we are talking and which one needs to be fixed. There are multiple rhythm-related experiments on production right now, but none of them are named _Who's got rhythm?_. I think it would be a good idea to have a dedicated textarea input for every Experiment in the admin dashboard that allows us to write down information, context, and updates related to that experiment. Perhaps even in markdown. This will make it easier for us to be on the same page when we are talking about experiments, or can make it easier to debug stuff when others are not around to supply us with the right information.

It could look something like this:

======================

## Internal notes

This is the Who's got rhythm experiment from August / September 2022, initiated by Person X and Person Y.

Contact people: Person X (Person Y does not work here anymore)
Results are analysed by: Person X and Person Z

### Important

- This experiment should go live in October 2022
- This experiment should be disabled after July 2025
- The slug cannot be changed as our AML blog links towards it.

### Logs:

2023-02-13, Person A: The Playlist has been updated with such and so because this and that happened, which required us to do this here and that there.

2023-04-18, Person C: I've updated the consent form with such and so in order to comply with the new regulations of this and that.

======================

**Comment by jaburgoyne** [comment]
We started a system loosely similar to this idea in the Researchers project board, although none of us has been great about keeping that fully up to date. I like the idea of putting this in the admin dashboard, so that it is all in one place.

Perhaps separately to this (or perhaps not), as a general rule, all slugs for 'major' experiments (i.e, anything that ended up features on the Amsterdam Music Lab page and thus the great majority of anything that makes it to the production server) should be perma-slugs. If we decide that an experiment has become so difficult to maintain that we wish to take it down, then the slug should still point to a page saying so. It's related to the broken-link issue on the web page in general, but this is a bit more specific to this issue.
